### ISSUE TITLE
Notification icon is ignored on OS X 10.9 and lower

### ISSUE THREAD

**Issue by miniak** [atom/brightray#169]
```
notification_presenter_mac.mm:

  if (base::mac::IsOSYosemiteOrLater() && !icon.drawsNothing())
    notification.contentImage = gfx::SkBitmapToNSImageWithColorSpace(icon, base::mac::GetGenericRGBColorSpace());
```

What is the reason for ignoring the icon on older versions of OS X?

**Comment by paulcbetts** [comment]
Older versions of OS X don't support `contentImage` at all

**Comment by miniak** [comment]
https://developer.apple.com/library/prerelease/mac/documentation/Foundation/Reference/NSUserNotification_Class/index.html#//apple_ref/occ/instp/NSUserNotification/contentImage

> @property(copy) NSImage *contentImage
> Available in OS X v10.9 and later.

**Comment by miniak** [comment]
I would rewrite it like this:
```
    if ([notification respondsToSelector:@selector(setContentImage:)] && !icon.drawsNothing())
        notification.contentImage = gfx::SkBitmapToNSImageWithColorSpace(icon, base::mac::GetGenericRGBColorSpace());
```

**Comment by miniak** [comment]
https://github.com/atom/brightray/pull/171

**Comment by paulcbetts** [comment]
Oops, off by one! Should be `IsOSMavericksOrLater`

**Comment by zcbenz** [comment]
Fixed by #171.
